integration/bundle/init_test.go

```diff
@@ -102,23 +99,22 @@ func TestBundleInitOnMlopsStacks(t *testing.T) {
 }
 
 func TestBundleInitHelpers(t *testing.T) {
-	env := testutil.GetEnvOrSkipTest(t, "CLOUD_ENV")
-	t.Log(env)
+	ctx, wt := acc.WorkspaceTest(t)
+	w := wt.W
 
-	w, err := databricks.NewWorkspaceClient(&databricks.Config{})
-	require.NoError(t, err)
-
-	me, err := w.CurrentUser.Me(context.Background())
+	me, err := w.CurrentUser.Me(ctx)
 	require.NoError(t, err)
 
 	var smallestNode string
-	switch env {
-	case "azure":
+	switch testutil.GetCloud(t) {
+	case testutil.Azure:
 		smallestNode = "Standard_D3_v2"
-	case "gcp":
+	case testutil.GCP:
 		smallestNode = "n1-standard-4"
-	default:
+	case testutil.AWS:
 		smallestNode = "i3.xlarge"
+	default:
+		t.Fatal("Unknown cloud environment")
```

Comment:
would be nice to include the value of testutil.GetCloud(t) into error message.

Reply:
Good point, will update in a later PR.